trex-operator/roles/server/templates/deployment.yml

```diff
@@ -55,6 +55,9 @@ spec:
 {% endif %}
             topologyKey: kubernetes.io/hostname
       serviceAccountName: trex-server-account
+{% if high_perf_runtime is defined and high_perf_runtime|length %}
+      runtimeClassName: "{{ high_perf_runtime }}"
```

Comment:
Same than testpmd-lb, I don't think we need this for trex since high_perf_runtime is not used in the TRex CR